index.html

```diff
@@ -17,15 +18,15 @@
       </nav>
     </header>
     <main>
-      <img src="https://via.placeholder.com/150x100" alt="SEDONA">
+      <img src="img/welcome.svg" width="456.65" height="350.88" alt="SEDONA">
```

Comment:
Значения `width` и `height` должны быть целыми числами. Дробные числа нужно округлять.
```suggestion
      <img src="img/welcome.svg" width="457" height="351" alt="SEDONA">
```